nixos/modules/services/mail/rmilter.nix

```diff
@@ -157,13 +176,9 @@ in
           type = types.str;
           description = "Addon to postfix configuration";
           default = ''
-smtpd_milters = ${head allSockets}
-# or for TCP socket
-# # smtpd_milters = inet:localhost:9900
-milter_protocol = 6
-milter_mail_macros = i {mail_addr} {client_addr} {client_name} {auth_authen}
-# skip mail without checks if milter will die
-milter_default_action = accept
```

Comment:
One more thing: I removed `milter_default_action` here because it would get appended last in my `main.cf` so I couldn't overwrite it anymore. This should be set in `services.postfix.extraConfig` manually if required IMHO. The postfix default is `tempfail` which should be fine.